src/components/UsageDataProvidersFieldArray/UsageDataProviderField.js

```diff
@@ -41,18 +41,40 @@ export default class UsageDataProviderField extends React.Component {
       onUDPSelected={this.props.onUDPSelected}
       renderTrigger={(props) => {
         this.triggerButton = props.buttonRef;
+        const UPDName = this.props.udp ? this.props.udp.label : '';
```

Comment:
```suggestion
        const UPDName = get(this.props.udp, 'label');
```

Reply:
This, and the conditional tooltip rendering changes are now done